utils.groovy

```diff
@@ -56,3 +56,9 @@ def getVersion() {
 }
 
 return this
+
+def jenkins_user_build_args() {
+  def jenkins_uid = sh (script: 'id -u jenkins', returnStdout: true).trim()
+  def jenkins_gid = sh (script: 'id -g jenkins', returnStdout: true).trim()
+  return " --build-arg JENKINS_UID=${jenkins_uid} --build-arg JENKINS_GID=${jenkins_gid}"
+}
```

Comment:
This has been moved here already by the [docs jenkinsfile PR](https://github.com/rstudio/rstudio-pro/pull/4196/files#diff-77cf514ee884c7a48b43dd41bab37b19c62e9ee3cd74a869f857670fb20baa87). We should only include it in one or the other. Also, wouldn't this need to be above the `return this` line?

Reply:
This ended up not being used for the windows build anyway. We can't use `sh` on windows.